controller/api/middleware.py

```diff
@@ -46,5 +46,6 @@ def process_response(self, request, response):
         a response header.
         """
         # clients shouldn't care about the patch release
-        response['X_DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]
+        response['DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]
+        response['X_DEIS_API_VERSION'] = __version__.rsplit('.', 1)[0]  # DEPRECATED
```

Comment:
Maybe have the old header assign its value from the new one, instead of finding the `rsplit` twice?